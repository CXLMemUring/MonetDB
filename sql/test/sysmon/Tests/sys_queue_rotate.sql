-- test the rotation of the queue
-- the case expression is used to make sure the test is timezone agnostic
select 1, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
select 2, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
select 3, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
select 4, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
select 5, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
select 6, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
