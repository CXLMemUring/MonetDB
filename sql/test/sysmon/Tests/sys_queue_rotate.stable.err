stderr of test 'sys_queue_rotate` in directory 'sql/test/sysmon` itself:


# 14:43:48 >  
# 14:43:48 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-28100" "--port=39321"
# 14:43:48 >  


# 14:43:48 >  
# 14:43:48 >  "Done."
# 14:43:48 >  

