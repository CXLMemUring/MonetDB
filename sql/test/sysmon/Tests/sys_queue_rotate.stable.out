stdout of test 'sys_queue_rotate` in directory 'sql/test/sysmon` itself:


# 14:46:47 >  
# 14:46:47 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-28342" "--port=32595"
# 14:46:47 >  

#select 1, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 1,	"monetdb",	"finished",	"select count(*) from sys.queue() where query = \\'select count(*) from sys.queue();\\';"	]
[ 1,	"monetdb",	"finished",	"set time..."	]
[ 1,	"monetdb",	"running",	"select 1, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
#select 2, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 2,	"monetdb",	"finished",	"set time..."	]
[ 2,	"monetdb",	"finished",	"select 1, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 2,	"monetdb",	"running",	"select 2, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
#select 3, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 3,	"monetdb",	"finished",	"select 1, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 3,	"monetdb",	"finished",	"select 2, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 3,	"monetdb",	"running",	"select 3, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
#select 4, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 4,	"monetdb",	"finished",	"select 2, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 4,	"monetdb",	"finished",	"select 3, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 4,	"monetdb",	"running",	"select 4, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
#select 5, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 5,	"monetdb",	"finished",	"select 3, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 5,	"monetdb",	"finished",	"select 4, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 5,	"monetdb",	"running",	"select 5, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
#select 6, username, status, case when query ilike 'set time%' then 'set time...' else query end from sys.queue();
% .%2,	.%1,	.%1,	.%3 # table_name
% %2,	username,	status,	%3 # name
% tinyint,	clob,	clob,	clob # type
% 1,	7,	8,	117 # length
[ 6,	"monetdb",	"finished",	"select 4, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 6,	"monetdb",	"finished",	"select 5, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]
[ 6,	"monetdb",	"running",	"select 6, username, status, case when query ilike \\'set time%\\' then \\'set time...\\' else query end from sys.queue();"	]

# 14:46:47 >  
# 14:46:47 >  "Done."
# 14:46:47 >  

